### PR TITLE
[netcore] Add a README.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

File: netcore/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# Running with netcore
+
+## Running through the 'dotnet' tool.
+
+Simply run ./dotnet <args>, it will use the mono runtime and corelib. Use the system dotnet
+for compilation etc.
+
+## Running with the mono runtime executable
+
+DYLD_LIBRARY_PATH=shared/Microsoft.NETCore.App/<dotnet version> MONO_PATH=shared/Microsoft.NETCore.App/<dotnet version> ../mono/mini/mono-sgen sample/HelloWorld/bin/netcoreapp3.0/HelloWorld.dll
```
Comment (lambdageek) [inline netcore/README.md]:
```suggestion
DYLD_LIBRARY_PATH=shared/Microsoft.NETCore.App/<dotnet version> MONO_PATH=shared/Microsoft.NETCore.App/<dotnet version> ../mono/mini/mono-sgen --assembly-loader=strict sample/HelloWorld/bin/netcoreapp3.0/HelloWorld.dll
```

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build Linux AArch64 FullAOT+Interpreter

**Comment by vargaz** [comment]
@monojenkins build Linux AArch64 FullAOT+Interpreter

**Pull request merged**